GUIDE.md

```diff
@@ -0,0 +1,129 @@
+# A Guide to `purescript-routing`
+
+A couple notes upfront:
```

Comment:
Looks ok to me: https://github.com/jasonzoladz/purescript-routing/blob/master/GUIDE.md